rust/src/treefile.rs

```diff
@@ -890,6 +890,19 @@ impl Treefile {
         n != map.len()
     }
 
+    pub(crate) fn remove_modules(&mut self, modules: Vec<String>, enable_only: bool) -> bool {
+        let modules_cfg = self.parsed.modules.ext_get_or_insert_default();
+        let map = if enable_only {
+            modules_cfg.enable.ext_get_or_insert_default()
+        } else {
+            modules_cfg.install.ext_get_or_insert_default()
+        };
+        let modules: BTreeSet<String> = modules.into_iter().collect();
```

Comment:
This is a very minor nit but I'd move this up to the top of the function so it's clear we're changing an input parameter format without any dependency on earlier bits.

Also, I would find it clearer if we called this e.g. `to_remove` instead of `modules` - the latter could be either the set of modules we need to remove, or the current one.

Reply:
Fixed in https://github.com/coreos/rpm-ostree/pull/3631!